### ISSUE TITLE
Get JPEG/GIF does not work on Embedded videos

### ISSUE THREAD

**Issue by hervaden** [WWBN/AVideo#7930]
**Describe the question**
Why doesn't the MP4ThumbnailsAndGif plugin does not retrieve the poster from a YouTube embedded video?  

The Logs show it is running, but the closed source nature does not allow me to debug.

**Screenshots**
![Screenshot 2023-04-21 at 11 06 02 AM](https://user-images.githubusercontent.com/59710655/233705069-7174cf56-d069-45f8-a19c-70b806c494fe.png)

![Screenshot 2023-04-21 at 11 06 09 AM](https://user-images.githubusercontent.com/59710655/233705097-0a4ce2eb-185a-467d-abe0-2494701c74dd.png)

**Comment by DanielnetoDotCom** [comment]
yes, there is no guarantee we can extract thumbs from embed videos.
